boxsdk/object/terms_of_service_user_status.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+from functools import partial
+import json
+
+from ..config import API
+from .base_object import BaseObject
+from ..util.translator import Translator
+
+
+class TermsOfServiceUserStatus(BaseObject):
+    """Represents a Box terms of service user status."""
+
+    _item_type = 'terms_of_service_user_status'
+
+    def get_url(self, *args):
+        return self._session.get_url('terms_of_service_user_statuses', self._object_id, *args)
+
+    def update(self, is_accepted):
```

Comment:
This seems like an unnecessary method. It's easy enough for someone to call

```tos_user_status.update_info(dict(is_accepted=is_accepted))```

You can keep it, I just don't think it's necessary.